### PR TITLE
Add Tables.rowcount

### PR DESCRIPTION
**Pull request by harsharora21**
This PR adds Tables.rowcount for TSFrames. It fixes issue #109

### CODE REVIEW

File: test/tables.jl [hunk 1]
```diff
@@ -13,6 +13,10 @@ ts = TSFrame(1:DAYS, dates)
 @test first(Tables.rows(ts))[:Index] == Date(YEAR, MONTH, 1)
 @test first(Tables.rows(ts))[:x1] == 1
 
+# testing Tables.rowcount
+@test Tables.rowcount(ts)==15
+@test Tables.rowcount(TSFrame(1:10))==10
+
```
Comment (chiraganand) [inline test/tables.jl:19]: Can you also add tests for boundary conditions with `0` and `1` rows respectively? Now, that there is a constructor for an empty `TSFrame` object then the `0` test becomes useful.

**Review by chiraganand — CHANGES_REQUESTED** [review]

File: test/tables.jl (hunk 1 above)
Comment (harsharora21) [reply test/tables.jl:23]: Added tests for 0 and 1 rows
